examples/99-advanced/atomic-orbitals.py

```diff
@@ -0,0 +1,224 @@
+#    ``opacity='linear'``. Instead, we use ``[1, 0, 1]`` since we would like
+#    the opacity to be proportional to the absolute value of the scalars.
+
+pl = pv.Plotter()
+vol = pl.add_volume(grid, cmap='magma', opacity=[1, 0, 1])
+vol.prop.interpolation_type = 'linear'
+pl.camera.zoom(1.5)
+pl.show_axes()
+pl.show()
+
+
+###############################################################################
+# Plot the Orbital Contours as an Isosurface
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Generate the contour plot for the orbital by determining when the orbital
+# equals 10% the maximum value of the orbital. This effectively captures the
+# majority of the "volume" the electron potentially exists in.
+#
+# Note how we use the absolute value of the scalars when evaluating
+# :func:`contour() <pyvista.PolyDataFilters.contour>`. Otherwise, we would only
+# capture the location of the spin-up electron.
+
+eval_at = grid['real_hwf'].max() * 0.1
+contours = grid.contour(
+    [eval_at],
+    scalars=np.abs(grid['real_hwf']),
+    method='marching_cubes',
+)
+contours = contours.interpolate(grid)
+contours.plot(
+    categories=2,
+    smooth_shading=True,
+    annotations={-eval_at: 'down-spin', eval_at: 'up-spin'},
+    scalar_bar_args={'n_labels': 0, 'title': ''},
+)
+
+
+###############################################################################
+# Volumetric Plot: Plot the Orbitals using RGBA
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Let's now combine some of the best parts of the two above plots. The
+# volumetric plot is great for showing the probability of the "electron cloud"
+# orbitals, but the colormap doesn't quite match reality as well as the
+# isosurface plot.
+#
+# For this example we're going to use an RGBA colormap to tightly control way
+# the orbitals are plotted. For this, the opacity will be mapped to the
+# probability of the electron being at a location in the grid, which we can do
+# by taking the absolute value of the orbital's "hydrogen wave function". We
+# can set the color of the orbital based on the phase, which we can get simply
+# by with ``orbital['real_hwf'] < 0``.
+#
+# Let's start with a simple one, the 3pz orbital.
+
+
+def plot_orbital(orbital, cpos='iso', clip_plane='x'):
+    """Plot an electron orbital using an RGBA colormap."""
+    neg_mask = orbital['real_hwf'] < 0
+    rgba = np.zeros((orbital.n_points, 4), np.uint8)
+    rgba[neg_mask, 0] = 255
+    rgba[~neg_mask, 1] = 255
+
+    # normalize opacity
+    opac = np.abs(orbital['real_hwf']) ** 2
+    opac /= opac.max()
+    rgba[:, -1] = opac * 255
+
+    orbital['plot_scalars'] = rgba
+
+    pl = pv.Plotter()
+    vol = pl.add_volume(
+        orbital,
+        scalars='plot_scalars',
+    )
+    vol.prop.interpolation_type = 'linear'
+    if clip_plane:
+        pl.add_volume_clip_plane(
+            vol, normal=clip_plane, normal_rotation=False, widget_color=pv.Color(opacity=0.0)
+        )
```

Comment:
It's kind of tough to use this without a bounding box. I was playing with it and quickly lost my volume

Reply:
Added bounding box back in 9811bbf.